src/SFA.DAS.FAT.Web/appsettings.json

```diff
@@ -11,6 +11,6 @@
   },
   "FindApprenticeshipTrainingApi": {
     "Key": "test",
-    "BaseUrl": "http://localhost:5003/"
+    "BaseUrl": "https://localhost:5003/"
```

Comment:
let's keep these changes in `appsettings.Development.json`

Reply:
changed back